chat-ui/src/services/conversations.ts

```diff
@@ -59,10 +51,17 @@ export class TimeoutError<Data extends object = object> extends Error {
   }
 }
 
+type ConversationServiceConfig = {
```

Comment:
Good idea to just export any type that is used in an exported public interface - in case users want to pre-construct the type or whatever
```suggestion
export type ConversationServiceConfig = {
```